gef-ui/component-tests/partials/amendments/eligibility.component-test.ts

```diff
@@ -0,0 +1,188 @@
+import { validationErrorHandler } from '../../../server/utils/helpers';
+import pageRenderer from '../../pageRenderer';
+import { EligibilityViewModel } from '../../../server/types/view-models/amendments/eligibility-view-model.ts';
+
+const page = 'partials/amendments/eligibility.njk';
+const render = pageRenderer(page);
+
+describe(page, () => {
+  const previousPage = 'previousPage';
+  const cancelUrl = 'cancelUrl';
+  const criteria = [
+    { id: 1, description: 'Test first criteria' },
```

Comment:
Can we please add an EC criterion with few bullet points i.e.

```text
1. Test criterions:
   * A
   * B
```

Reply:
Now added! Here's a screen shot of what it will look like now (test input):

<img width="400" alt="image" src="https://github.com/user-attachments/assets/fae9a2df-9373-4b49-876f-a1f2eb2772dc" />

Reply:
I think the GEF ECs are `<ul>`, could you please check GEF EC to ensur ethe design is the same. If they are same then please let me know I'll resolve this comment.

Reply:
Good check, they are `<ol>` so consistent with how it's currently implemented. Here's a screenshot of the GEF ECs on the site: 

<img width="889" alt="image" src="https://github.com/user-attachments/assets/1f53a8b1-1af8-4953-b35f-1961880f6a21" />

Reply:
Thank you so much @BethThomas141.